src/offset/mod.rs

```diff
@@ -121,6 +121,24 @@ impl<T> MappedLocalTime<T> {
             MappedLocalTime::Ambiguous(min, max) => MappedLocalTime::Ambiguous(f(min), f(max)),
         }
     }
+
+    /// Maps a `MappedLocalTime<T>` into `MappedLocalTime<U>` with given function.
+    ///
+    /// Returns `MappedLocalTime::None` if the function returns `None`.
+    #[must_use]
+    pub(crate) fn try_map<U, F: FnMut(T) -> Option<U>>(self, mut f: F) -> MappedLocalTime<U> {
```

Comment:
I think the `Option` variant of this is called `and_then()`, maybe reuse that name?

Reply:
:+1: `and_then()` is better.